card/db/migrate_core_cards/20150220134731_following_to_follow_rule.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+
+class FollowingToFollowRule < Card::CoreMigration
+  def up
+    Card.search(:right=>{:codename=>'following'}, :left=>{:type=>'user'}).each do |following_card|
+      user_name = following_card.cardname.left
+      following_card.item_names.each do |followed_set_name|
+        
+        set_card = Card.fetch(follwed_set_name, :new=>{})
+        if (!set_card.type_code == :set)
+          set_card = set_card.default_follow_set_card
+        end
+        rule = Card.fetch set_card.follow_rule_name(user_name), :new=>{:type=>'pointer'}
+        rule.content = "[[always]]"
```

Comment:
I suspect this approach is probably fine, as it's unlikely for it to exist, and it's possible for it to exist and not cause a problem if updated this way.

I should note that it can really cause headaches when we raise errors over minor problems in content migration.  It kills the whole migration and reverts everything it's done, often for pretty minor errors that would not otherwise prevent the site from running normally.  This is particularly a problem on Cloudstore, when we're usually trying to get through a bunch of migrations at once, and tracking down individual migration errors can get tricky.

I think what I would suggest is that we support an optionally separate "migration errors" log for both fatal and non-fatal migration errors.  As a maintainer of lots of wagn sites, this would be a big help.

None of this should block this PR!